core-bundle/src/Command/ResizeImagesCommand.php

```diff
@@ -211,7 +211,7 @@ private function executeInCurrentProcess(float $timeLimit, float $concurrent, bo
                 if ($timeLimit && microtime(true) - $startTime + $sleep > $timeLimit) {
                     $this->io->writeln('Time limit of '.$timeLimit.' seconds reached.');
 
-                    return 0;
+                    break;
```

Comment:
@ausi Can you double check this is correct? When breaking out of the foreach loop we should now IMHO hit the `return` on L244 if L237 does not evaluate to `true`.

Reply:
As the only code after the foreach is the `return 0;` line I’m pretty sure this change has the exact behavior as before. The `finally` block gets executed with both versions.

Why does phpstan want to change this to `break`?

Reply:
> Why does phpstan want to change this to break?

Because the `return 1` from the `finally` block would overwrite this return which is supposedly bad practice (My PHPStorm complains about this as well.). It's at least a bit hard to understand I guess. :upside_down_face: 

The exact message from PHPStan was: `This return is overwritten by a different one in the finally block below.`.

Reply:
> This return is overwritten by a different one in the finally block below.

That’s the beauty of the `finally` block to me ☺️
But I agree, a `break` here is better for understanding the code correctly.